CONTRIBUTING.md

```diff
@@ -188,7 +194,7 @@ on _Details_ to find out more about them.
 
 ### Updating the component API documentation
 
-The component API in the component `propTypes` and under `docs/pages/api-docs` is auto-generated from the JSDOC in the TypeScript declarations.
+The component API in the component `propTypes` and under `docs/pages/api-docs` is auto-generated from the [JSDOC](https://jsdoc.app/about-getting-started.html) in the TypeScript declarations.
```

Comment:
This seems to be the official capitalization:
```suggestion
The component API in the component `propTypes` and under `docs/pages/api-docs` is auto-generated from the [JSDoc](https://jsdoc.app/about-getting-started.html) in the TypeScript declarations.
```

Reply:
let's do this in another PR.